controllers/designate_controller.go

```diff
@@ -1059,9 +1059,55 @@ func (r *DesignateReconciler) generateServiceConfigMaps(
 	// - %-config configmap holding minimal designate config required to get the service up, user can add additional files to be added to the service
 	// - parameters which has passwords gets added from the ospSecret via the init container
 	//
+	Log := r.GetLogger(ctx)
 
 	cmLabels := labels.GetLabels(instance, labels.GetGroupLabel(designate.ServiceName), map[string]string{})
+	replicas := int(*instance.Spec.DesignateBackendbind9.Replicas)
+	secretName := designate.DesignateBindKeySecret
```

Comment:
nit: where. this is only used once you don't really need a temp var for it.

Reply:
ok, I will remove the secretName variable, thanks.